fedimint-core/src/db/mod.rs

```diff
@@ -2184,6 +2184,31 @@ pub type CoreMigrationFn = for<'r, 'tx> fn(
     Box<maybe_add_send!(dyn futures::Future<Output = anyhow::Result<()>> + 'r)>,
 >;
 
+/// Verifies that all database migrations are defined contiguously and returns
+/// the "current" database version, which is one greater than the last key in
+/// the map.
+pub fn get_current_database_version<F>(
+    migrations: &BTreeMap<DatabaseVersion, F>,
+) -> DatabaseVersion {
+    let mut versions = migrations.keys().cloned().collect::<Vec<_>>();
+    versions.sort();
```

Comment:
Isn't btreemap already sorted?

Reply:
Didn't know if collect would guarantee the order, but quick googles indicates that its already sorted. Will remove `sort`